pkg/kubelet/cm/topologymanager/topology_manager_test.go

```diff
@@ -93,155 +140,6 @@ func (m *mockHintProvider) Allocate(pod *v1.Pod, container *v1.Container) error
-		mngr := manager{}
-		actual := mngr.GetAffinity(tc.podUID, tc.containerName)
-		if !reflect.DeepEqual(actual, tc.expected) {
-			t.Errorf("Expected Affinity in result to be %v, got %v", tc.expected, actual)
-		}
-	}
-}
-
-func TestAccumulateProvidersHints(t *testing.T) {
-	tcases := []struct {
-		name     string
-		hp       []HintProvider
-		expected []map[string][]TopologyHint
-	}{
-		{
-			name:     "TopologyHint not set",
-			hp:       []HintProvider{},
-			expected: nil,
-		},
-		{
-			name: "HintProvider returns empty non-nil map[string][]TopologyHint",
-			hp: []HintProvider{
-				&mockHintProvider{
-					map[string][]TopologyHint{},
-				},
-			},
-			expected: []map[string][]TopologyHint{
-				{},
-			},
-		},
-		{
-			name: "HintProvider returns - nil map[string][]TopologyHint from provider",
-			hp: []HintProvider{
-				&mockHintProvider{
-					map[string][]TopologyHint{
-						"resource": nil,
-					},
-				},
-			},
-			expected: []map[string][]TopologyHint{
-				{
-					"resource": nil,
-				},
-			},
-		},
-		{
-			name: "2 HintProviders with 1 resource returns hints",
-			hp: []HintProvider{
-				&mockHintProvider{
-					map[string][]TopologyHint{
-						"resource1": {TopologyHint{}},
-					},
-				},
-				&mockHintProvider{
-					map[string][]TopologyHint{
-						"resource2": {TopologyHint{}},
-					},
-				},
-			},
-			expected: []map[string][]TopologyHint{
-				{
-					"resource1": {TopologyHint{}},
-				},
-				{
-					"resource2": {TopologyHint{}},
-				},
-			},
-		},
-		{
-			name: "2 HintProviders 1 with 1 resource 1 with nil hints",
-			hp: []HintProvider{
-				&mockHintProvider{
-					map[string][]TopologyHint{
-						"resource1": {TopologyHint{}},
-					},
-				},
-				&mockHintProvider{nil},
-			},
-			expected: []map[string][]TopologyHint{
-				{
-					"resource1": {TopologyHint{}},
-				},
-				nil,
-			},
-		},
-		{
-			name: "2 HintProviders 1 with 1 resource 1 empty hints",
-			hp: []HintProvider{
-				&mockHintProvider{
-					map[string][]TopologyHint{
-						"resource1": {TopologyHint{}},
-					},
-				},
-				&mockHintProvider{
-					map[string][]TopologyHint{},
-				},
-			},
-			expected: []map[string][]TopologyHint{
-				{
-					"resource1": {TopologyHint{}},
-				},
-				{},
-			},
-		},
-		{
-			name: "HintProvider with 2 resources returns hints",
-			hp: []HintProvider{
-				&mockHintProvider{
-					map[string][]TopologyHint{
-						"resource1": {TopologyHint{}},
-						"resource2": {TopologyHint{}},
-					},
-				},
-			},
-			expected: []map[string][]TopologyHint{
-				{
-					"resource1": {TopologyHint{}},
-					"resource2": {TopologyHint{}},
-				},
-			},
-		},
-	}
-
-	for _, tc := range tcases {
-		mngr := manager{
-			hintProviders: tc.hp,
-		}
-		actual := mngr.accumulateProvidersHints(&v1.Pod{}, &v1.Container{})
-		if !reflect.DeepEqual(actual, tc.expected) {
-			t.Errorf("Test Case %s: Expected NUMANodeAffinity in result to be %v, got %v", tc.name, tc.expected, actual)
-		}
-	}
-}
-
```

Comment:
I guess all of these removed tests were moved to scope-level tests, since that is where the logic lives now?
No need to make the change at this point, but it would have been nice to see the move in a single commit to verify that nothing was forgotten / deleted.

Reply:
For future reference I probably would have broken this down as:
1. Reposition top-level topology manager tests as scope-level tests
2. Rename scope-level tests to prepend `container` in the name
3. Add `pod`-level scope tests
4. Add top-level scope tests to topology-manager